docs/source/conf.py

```diff
@@ -44,6 +45,20 @@ def get_authors_from_citation_file(file_name):
     return author[:-2]
 
 
+def get_python_version_from_pyproject():
+    """
+    Read python version from pyproject.toml file
+
+    """
+    with open("../../pyproject.toml") as file:
```

Comment:
Relative path are taken from the current working directory, not the current file. So to make this work more generally, use `Path(__file__).parent / "../../pyproject.toml"`